release/release_logs/2.5.0/stress_tests/stress_test_many_tasks.json

```diff
@@ -0,0 +1,47 @@
+{
+    "perf_metrics": [
+        {
+            "perf_metric_name": "stage_0_time",
+            "perf_metric_type": "LATENCY",
+            "perf_metric_value": 9.974027872085571
+        },
+        {
+            "perf_metric_name": "stage_1_avg_iteration_time",
+            "perf_metric_type": "LATENCY",
+            "perf_metric_value": 21.81450412273407
+        },
+        {
+            "perf_metric_name": "stage_2_avg_iteration_time",
+            "perf_metric_type": "LATENCY",
+            "perf_metric_value": 55.80217342376709
+        },
+        {
+            "perf_metric_name": "stage_3_creation_time",
+            "perf_metric_type": "LATENCY",
+            "perf_metric_value": 4.1170501708984375
+        },
+        {
+            "perf_metric_name": "stage_3_time",
+            "perf_metric_type": "LATENCY",
+            "perf_metric_value": 2812.0461761951447
+        },
+        {
+            "perf_metric_name": "stage_4_spread",
+            "perf_metric_type": "LATENCY",
+            "perf_metric_value": 0.6432904141188599
+        }
+    ],
+    "stage_0_time": 9.974027872085571,
+    "stage_1_avg_iteration_time": 21.81450412273407,
+    "stage_1_max_iteration_time": 22.599802017211914,
+    "stage_1_min_iteration_time": 21.001055240631104,
+    "stage_1_time": 218.14512729644775,
+    "stage_2_avg_iteration_time": 55.80217342376709,
+    "stage_2_max_iteration_time": 56.71760582923889,
+    "stage_2_min_iteration_time": 53.68144226074219,
+    "stage_2_time": 279.0118091106415,
+    "stage_3_creation_time": 4.1170501708984375,
+    "stage_3_time": 2812.0461761951447,
```

Comment:
was 2541

Reply:
This might be a real regression given it's e2e but I took a glance, doesn't seem there's a single run that caused it...